### PR TITLE
Merge 177x into develop 19/08/2020

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Merge 1.7.7.x into develop
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | ~
| How to test?  | ~

Conflicts
```
	both modified:   admin-dev/themes/default/template/footer.tpl
	both modified:   composer.lock
	both modified:   tests/UI/pages/BO/international/currencies/index.js
	both modified:   tests/UI/pages/BO/shopParameters/trafficAndSeo/seoAndUrls/index.js
	both modified:   themes/_core/js/product.js
	deleted by us:   themes/core.js
	deleted by us:   themes/core.js.map
```

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20624)

### CODE REVIEW

**Comment by matks** [comment]
@NeOMakinG Could you check file themes/_core/js/product.js ? I had multiple git conflicts on it 😭

File: themes/_core/js/product.js [hunk 1]
```diff
@@ -191,23 +191,43 @@ function updateProduct(event, eventType, updateUrl) {
...
+          .first()
+          .replaceWith(data.product_additional_info);
+        $('.quickview #product-details, #product-details').replaceWith(data.product_details);
+        $(
+          '.quickview .product-flags, .page-product:not(.modal-open) .row .product-flags, .page-product:not(.modal-open) .product-container .product-flags',
+        )
+          .first()
+          .replaceWith(data.product_flags);
+        replaceAddToCartSections(data);
+        const minimalProductQuantity = parseInt(data.product_minimal_quantity, 10);
```
Comment (NeOMakinG) [inline themes/_core/js/product.js]: You choosed the wrong one, you should keep selectors map in place

**Review by NeOMakinG — CHANGES_REQUESTED** [review]

File: themes/_core/js/product.js (hunk 1 above)
Comment (matks) [reply themes/_core/js/product.js]: It was fifty/fifty 😭

Comment (matks) [reply themes/_core/js/product.js]: Fixed, I believe, in https://github.com/PrestaShop/PrestaShop/pull/20624/commits/ae4811076c8e407db5100ec89a08502a2f2648df

**Review by NeOMakinG — APPROVED** [review]
Looks fine for me

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**